vtex/omnishipping
# Fixed Receiver Name wrongly being removed if it is not changed

#### What is the purpose of this pull request?
Fixed Receiver Name wrongly being removed if new address does not have `receiverName` but user already inserted personal info

#### What problem is this solving?
Receiver name was being removed when a new address was inserted.

#### How should this be manually tested?
1. Add [items to cart](https://fernando--vtexgame1.myvtex.com/checkout/cart/add?&sku=285&qty=1&seller=1&sc=1&sku=289&qty=1&seller=1&sc=1)
2. Go to shipping and select pickup point
3. Search for a pickup point in `Praia de botafogo 300`
4. Go to remaining items and type `22072060`
5. Receiver name should be auto completed

#### Screenshots or example usage
n/a

#### Types of changes
